src/istgt_lu_disk.c

```diff
@@ -6815,7 +6815,11 @@ istgt_lu_disk_status(ISTGT_LU_Ptr lu, int lun)
 		return -1;
 
 	MTX_LOCK(&spec->state_mutex);
-	status = spec->state;
+	MTX_LOCK(&spec->rq_mtx);
```

Comment:
why this change?

Reply:
With this change, In `istgtcontrol status`, if enough number of replicas are not connected to spec then we need to print volume status as `FAKE`.